read-fonts/src/tables/glyf.rs

```diff
@@ -678,6 +682,19 @@ impl fmt::Display for ToPathError {
     }
 }
 
+/// The order to process points in a glyf point stream is ambiguous when the first point is
+/// off-curve. Major implementations differ. Which one would you like to match?
+#[derive(Debug, Default, Copy, Clone)]
+pub enum ToPathStyle {
+    /// If the first point is off-curve, check if the last is on-curve
+    /// If it is, start there. If it isn't, start at the implied midpoint between first and last.
+    #[default]
+    FreeType,
+    /// If the first point is off-curve, check if the second is on-curve.
+    /// If it is, start there. If it isn't, start at the implied midpoint between first and second.
+    HbDraw,
```

Comment:
optional but I feel like `HarfBuzz` is more easily recognizable than `HbDraw` even if it's slightly less specific.

Reply:
Good call, will rename.